approval_polls/views.py

```diff
@@ -12,6 +12,13 @@
 
 def index(request):
   poll_list = Poll.objects.filter(pub_date__lte=timezone.now()).order_by('-pub_date')
+  return GetPolls(request, poll_list, 'approval_polls/index.html')
+
+def PollsView(request):
```

Comment:
Should be protected by the `@login_required` decorator. Only authenticated users should be able to access this page. 
Also, `PollsView` can be renamed to `myPolls`